src/components/Homepage/caseStudy.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import { Row, Col } from '../grid'
+import breakpoint from 'styled-components-breakpoint'
+import { Margin, Padding } from 'styled-components-spacing'
+import StyledLink from '../styledLink'
+import Image from '../Common/Image'
+import { H2, Paragraph } from '../Typography'
+import getIntroSentence from '../../utils/getIntroSentence'
+
+const ImageWrapper = styled(Col)`
+  margin-top: ${remcalc(-5)};
+  max-width: 100%;
+  height: auto;
+
+  img {
+    max-width: 100%;
+    width: ${remcalc(526)};
+
+    ${breakpoint('desktop')`
+      max-width: 100%;
+      width: ${remcalc(526)};
+    `}
+  }
+`
+
+const WrapperRow = styled(Row)``
+
+const SubHeading = styled(Paragraph)`
+  color: #757575;
+  padding-bottom: 0;
+  opacity: 0.7;
+`
+
+const TextWrapper = styled.div`
+  ${breakpoint('tablet')`
+    align-self: center;
+    width: 431px;
+  `}
+`
+
+const FlexCol = styled(Col)`
```

Comment:
we can also pass block={false} to our `Col` component. (I don't know why this isn't the default prop btw)

Reply:
I removed it because being flex didn't actually do anything

Reply:
xD